gradle-baseline-java/src/main/java/com/palantir/baseline/plugins/BaselineModuleJvmArgs.java

```diff
@@ -376,9 +381,10 @@ public final Iterable<String> asArguments() {
 
             log.debug(
                     isCompilation
-                            ? "BaselineModuleJvmArgs compiling on {} with exports: {}"
-                            : "BaselineModuleJvmArgs executing task on project {} with exports: {}",
-                    getProjectPath(),
+                            ? "BaselineModuleJvmArgs compiling {} on project {} with exports: {}"
+                            : "BaselineModuleJvmArgs executing {} on project {} with exports: {}",
+                    getTaskName().getOrNull(),
+                    getProjectPath().getOrNull(),
```

Comment:
Do we really need this `isCompilation` conditional? It seems a bit confusing to log something like "compiling 'compileJava'`. It should be pretty obvious from the task name whether it's a compilation task or not.